banyand/measure/query.go

```diff
@@ -128,6 +140,28 @@ func (s *measure) Query(ctx context.Context, mqo pbv1.MeasureQueryOptions) (pbv1
 	if tstIter.Error() != nil {
 		return nil, fmt.Errorf("cannot iterate tstIter: %w", tstIter.Error())
 	}
+
+	entities := make([]string, 0)
+	if s.GetSchema().GetTagFamilies()[0].GetName() == topNTagFamily {
+		for _, topNAggregation := range s.topNAggregations {
+			if topNAggregation.GetMetadata().GetName() == s.schema.GetMetadata().GetName() {
+				entities = append(entities, "sortDirection", "rankNumber")
```

Comment:
The entity used to define the topN result measure lacks tags, causing a mismatch between writing and reading. https://github.com/apache/skywalking-banyandb/blob/main/banyand/measure/metadata.go#L214 is relevant. 

Please correct this issue, please prepend "sortDirection" and "rankNumber" to the above Entity field.